js/interface.js

```diff
@@ -349,6 +433,13 @@
     });
   }
 
+  function deleteInstances(options) {
```

Comment:
I don't think you have tested this function in all the places where it's used because in its current form it's impossible that it'll always work.

You used `deleteInstances()` in one place where the input parameter is an instance object. And then in another place, you used the function where the input parameter is an array of IDs. They can't both work.